src/protobuf_stuff/google/protobuf/duration.ts

```diff
@@ -26,7 +26,7 @@ export const protobufPackage = "google.protobuf";
  *     if (duration.seconds < 0 && duration.nanos > 0) {
  *       duration.seconds += 1;
  *       duration.nanos -= 1000000000;
- *     } else if (duration.seconds > 0 && duration.nanos < 0) {
+ *     } else if (durations.seconds > 0 && duration.nanos < 0) {
```

Comment:
little typo in the example, why'd it change to plural?

Reply:
It's taken from the cosmos-sdk protobuf comments :shrug: